src/v2/view-builder/views/password/EnrollAuthenticatorPasswordView.js

```diff
@@ -47,8 +47,7 @@ const Body = BaseForm.extend({
   },
 
   triggerAfterError(model, error) {
-    const policy = this.getPasswordPolicySettings();
-    error.responseJSON = removeRequirementsFromError(error.responseJSON, policy);
+    error.responseJSON = removeRequirementsFromError(error.responseJSON);
```

Comment:
We don't need to follow the exact v1 way for this (removing the requirements one by one) anymore since i18n lies in SIW only. We can just get the message directly from properties.